ui/src/test/java/edu/wpi/grip/ui/codegeneration/GripIconHSLSetup.java

```diff
@@ -0,0 +1,53 @@
+package edu.wpi.grip.ui.codegeneration;
+
+import edu.wpi.grip.core.OperationMetaData;
+import edu.wpi.grip.core.Step;
+import edu.wpi.grip.core.operations.composite.HSLThresholdOperation;
+import edu.wpi.grip.core.sockets.InputSocket;
+import edu.wpi.grip.core.sockets.OutputSocket;
+import edu.wpi.grip.core.sources.ImageFileSource;
+import edu.wpi.grip.util.Files;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GripIconHSLSetup {
+  public static List<Number> defaultHVal;
+  public static List<Number> defaultSVal;
+  public static List<Number> defaultLVal;
+
+  static {
+    defaultHVal = new ArrayList<Number>();
+    defaultSVal = new ArrayList<Number>();
+    defaultLVal = new ArrayList<Number>();
+    defaultHVal.add(new Double(0.0));
+    defaultHVal.add(new Double(49.0));
+    defaultSVal.add(new Double(0.0));
+    defaultSVal.add(new Double(41.0));
+    defaultLVal.add(new Double(0.0));
+    defaultLVal.add(new Double(67.0));
```

Comment:
`Arrays.asList(0.0d);`?? This would be way better.
Also, you could do `Collections.singletonList(0.0d)`